docs/kubernetes.mdx

```diff
@@ -20,6 +20,19 @@ minikube start --driver=virtualbox
 
 :::
 
+:::tip
+
+To quickly try out the Parca and Parca Agent with Kubernetes on Apple Silicon machine (ARM architecture), you create a [Colima](https://github.com/abiosoft/colima) cluster with Kubernetes:
```

Comment:
if I understand correctly colima works on all platforms, any reason we shouldn't just have it be the only recommendation?